src/getSuggestions.js

```diff
@@ -0,0 +1,22 @@
+import { memoize } from 'cerebro-tools'
+
+/**
+ * Get google suggestions for entered query
+ * @param  {String} query
+ * @return {Promise}
+ */
+const getSuggestions = (query) => {
+  const url = `https://duckduckgo.com/ac/?q=${query}`
```

Comment:
@KELiON omg! I was still reading DuckDuckGo's docs and you already made this as a working feature. Incredible! 💯